allennlp/tests/fixtures/elmo_port/shards/shard0

```diff
@@ -0,0 +1,100 @@
+The U.S. Centers for Disease Control and Prevention initially advised school systems to close if outbreaks occurred , then reversed itself , saying the apparent mildness of the virus meant most schools and day care centers should stay open , even if they had confirmed cases of swine flu .
```

Comment:
do we still want to call this directory `elmo_port`? or should we call it something more general like `language_modeling`

Reply:
Done. To some extent it might be helpful to explain where the data is coming from, but I'm not sure this matters much for fixtures.